packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-conf

```diff
@@ -18,7 +20,15 @@ flags:
   nvidiaDriverRoot: "/"
   plugin:
     passDeviceSpecs: {{default true settings.kubelet-device-plugins.nvidia.pass-device-specs}}
-    deviceListStrategy: {{default "volume-mounts" settings.kubelet-device-plugins.nvidia.device-list-strategy}}
+    {{#if settings.kubelet-device-plugins.nvidia.device-list-strategy}}
+    {{#if (eq settings.kubelet-device-plugins.nvidia.device-list-strategy "cdi-cri")}}
+    deviceListStrategy: "cdi-cri"
+    {{else}}
+    deviceListStrategy: {{settings.kubelet-device-plugins.nvidia.device-list-strategy}}
+    {{/if}}
+    {{else}}
+    deviceListStrategy: "volume-mounts"
+    {{/if}}
```

Comment:
You don't have to conditionally setting the `cdi-cri` value. You can just keep the line as it was before, since we control the CDI support in the NVIDIA Container Runtime configuration.